pkg/client/client.go

```diff
@@ -50,10 +51,15 @@ func (c *Client) BuildConfigs(namespace string) BuildConfigInterface {
 }
 
 // BuildLogs provides a REST client for BuildLogs
-func (c *Client) BuildLogs(namespace string) BuildLogInterface {
+func (c *Client) BuildLogs(namespace string) BuildLogsInterface {
 	return newBuildLogs(c, namespace)
 }
 
+// BuildGenerate provides a REST client for Build generation
+func (c *Client) BuildGenerate(namespace string) BuildGenerateInterface {
```

Comment:
Method on the client being "Generate" is fine, my concern was w/ calling the interface itself "BuildGenerateInterface" (should be BuildGeneratorInterface)